README.md

```diff
@@ -5,7 +5,7 @@
 
 A sane client for web services with a built-in circuit-breaker, support for filtering both request and response.
 ```
-npm install perron --save
+yarn install perron --save
```

Comment:
I think this PR is mainly for development because the lock file is not used by library users. Users of the library can still keep using `npm`. How about leaving `npm install perron --save` and add `yarn add perron` ([`yarn install` is for installing all dependencies](https://yarnpkg.com/lang/en/docs/cli/install/))?